GHC 8.6 compatibilty, version bump

0xCM opened the pull request.

0xCM committed:
GHC 8.6 compatibilty, version bump

conklech committed:
Bump dependencies; update travis

conklech commented:
Thanks! Are there any other changes you'd like before I push a new release to Hackage (once the build passes)?

conklech committed:
Tweak .travis.yml to drop nightly

vector-space is not in the current nightly (as of this writing nighly recently bumped GHC versions), and given our general release cycle it's not necessary to test against nightly.

0xCM commented:
No, I'm good, thanks!
